docs/components/plugins/APIBox.tsx

```diff
@@ -0,0 +1,21 @@
+import { PlatformName } from '../../types/common';
+
+import { H3 } from '~/components/plugins/Headings';
+import { PlatformTags } from '~/components/plugins/PlatformTag';
+import { STYLES_APIBOX } from '~/components/plugins/api/APISectionUtils';
```

Comment:
```suggestion
import { H3 } from '~/components/plugins/Headings';
import { PlatformTags } from '~/components/plugins/PlatformTag';
import { STYLES_APIBOX } from '~/components/plugins/api/APISectionUtils';
import { PlatformName } from '~/types/common';
```

Let's unify import convention.